isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/anwendungskern.adoc

```diff
@@ -0,0 +1,201 @@
+= Anwendungskern
+
+include::documentation-guide::partial$licence.adoc[]
+
+// tag::anwendungskern-definition[]
+
+Der xref:glossary::terms-definitions.adoc#glossar-anwendungskern[Anwendungskern] ist der zentrale Bestandteil eines Backends.
+Hier werden die fachlichen Funktionen der zugehörigen xref:glossary::terms-definitions.adoc#glossar-anwendung[Anwendung] realisiert.
+Der Aufbau des Anwendungskerns sowie seine Kommunikation mit anderen Schichten ist in <<systemarchitektur-anwendungskern,folgender Grafik>> dargestellt.
+
+[[systemarchitektur-anwendungskern]]
+.Einbettung des Anwendungskerns in die Systemarchitektur
+image::referenzarchitektur:software-technisch/backend/anwendungskern.dn.svg[width=80%,align=center]
+
+// end::anwendungskern-definition[]
+
+[[fachkomponenten]]
+== Fachkomponenten
+
+// tag::fachkomponente-definition[]
+
+*Fachkomponenten* setzen die Anwendungskomponenten aus der fachlichen Architektur um.
+Sie bilden damit den Schnitt der Anwendungskomponenten in die technische Architektur ab.
```

Comment:
Dass Fachkomponenten den "Schnitt von Anwendungskomponenten in eine technische Architektur" abbilden würden, wäre mir neu. Kennt denn die Fachkomponente die technische Architektur? Eher würde ich sagen, sie sollte unabhängig von ihr sein.

Reply:
Die Fachkomponenten **gehören** durch ihre Definition zur technischen Architektur. Die Definition enthält auch, dass sie einen möglichst geringen Bezug zur Technik haben.